decentraland/unity-renderer
# chore: Remove FPS cap toggle

## What does this PR change?

It removes the FPS cap toggle from the Graphics sections of Settings. It also deletes the functionality related to the toggle, since according to this [PR](https://github.com/decentraland/unity-renderer/pull/2422) and this [investigation](https://docs.google.com/document/d/1nH5Jd2T8Mfk2IaS1JqVLKEb3dROfBqcNgp8u6Y9tb4E/edit) it wont be needed anymore.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=chore/remove-FPS-cap-toggle
2. Open Settings->Graphics
3. Check that the FPS Caps toggle isnt there

![image](https://user-images.githubusercontent.com/1999557/177647761-66df3157-3f67-434c-be1c-fd0556fb8b3e.png)


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
